pkg/ami/auto_resolver.go

```diff
@@ -44,9 +44,6 @@ func MakeImageSearchPatterns(version string) map[string]map[int]string {
 		api.NodeImageFamilyWindowsServer2004CoreContainer: {
```

Comment:
From the docs looks like Windows Server 2004 Core also reached the end of support on 12/14/2021, shall we remove that too? 🤔

Reply:
Yup, `NodeImageFamilyWindowsServer2004CoreContainer` also needs to be removed as it's no longer supported. Attempting to use it will result in an error.

Reply:
Thanks both! Removed `WindowsServer2004CoreContainer` in the latest commit. :)